src/unity/python/turicreate/toolkits/sound_classifier/sound_classifier.py

```diff
@@ -61,18 +118,17 @@ def create(dataset, target, feature, max_iterations=10, verbose=True,
         If you are getting memory errors, try decreasing this value. If you
         have a powerful computer, increasing this value may improve performance.
     '''
-    from ._audio_feature_extractor import _get_feature_extractor
-    import mxnet as _mx
-    import time as _time
+    import time as time
```

Comment:
Just a general question: Is it better practice to use the `as` keyword even if we don't want to use a shorthand for the package?

Reply:
This was a search and replace error. I've fixed it.